### PR TITLE
fix(ci): properly set helm chart version

### PR DESCRIPTION
**Pull request by olevski**
This should fix the previous mess I made.

### CODE REVIEW

**Comment by olevski** [comment]
Example of working workflow. I commented out the part that actually does the publishing. But you can see all the other parts that figure out the tag - and it comes up with the right tag.

https://github.com/SwissDataScienceCenter/renku/runs/5093125960?check_suite_focus=true

File: .github/workflows/publish-master-merges.yaml [hunk 1]
```diff
@@ -17,19 +17,28 @@ jobs:
...
+        with:
+          version: '3.8.0'
+        id: setup-helm
+      - id: get-latest-helm-charts
+        run: |
+          helm repo add renku https://swissdatasciencecenter.github.io/helm-charts/
+          helm repo update
+          helm search repo renku --devel -o yaml > .latest-charts.yaml
+      - id: get-latest-version
+        uses: mikefarah/yq@v4.19.1
```
Comment (rokroskar) [inline .github/workflows/publish-master-merges.yaml:30]: I wonder if `jq` is present in the actions builders already by default - if that's the case I would skip this extra dependency and output to json in the step above.

Comment (olevski) [reply .github/workflows/publish-master-merges.yaml:30]: Ok I will test for that.

Comment (olevski) [reply .github/workflows/publish-master-merges.yaml]: yeah that works, see here: https://github.com/SwissDataScienceCenter/renku/runs/5093411722?check_suite_focus=true

One sec let me just remove the stuff I added to test

**Review by ableuler — APPROVED** [review]
LGTM

File: .github/workflows/publish-master-merges.yaml (hunk 1 above)
Comment (rokroskar) [reply .github/workflows/publish-master-merges.yaml]: looks good, thanks!

**Pull request merged**